xmtp_configuration/src/common/mls.rs

```diff
@@ -40,4 +40,4 @@ pub const SUPER_ADMIN_METADATA_PREFIX: &str = "_";
 pub const HMAC_SALT: &[u8] = b"libXMTP HKDF salt!";
 
 pub const ENABLE_COMMIT_LOG: bool = true;
-pub const MIN_RECOVERY_REQUEST_VERSION: &str = "1.6.1";
+pub const MIN_RECOVERY_REQUEST_VERSION: &str = "1.6.0";
```

Comment:
setting this back to 1.6.0 until we update root `Cargo.toml` version to 1.6.1 during release process, in order to allow tests to pass